editor/create_dialog.cpp

```diff
@@ -255,8 +255,9 @@ void CreateDialog::_update_search() {
 		if (base_type == "Node" && type.begins_with("Editor"))
 			continue; // do not show editor nodes
 
-		if (base_type == "Resource" && ClassDB::is_parent_class(type, "Script"))
-			continue; // do not show script nodes
+		if (base_type == "Resource" && ClassDB::is_parent_class(type, "PluginScript"))
```

Comment:
This is not gonna work, this prevent PluginScript children to be displayer but not PluginScript, which is the one who causes the crash, proper sentence would be:
> if (base_type == "Resource" && type == "PluginScript")

Reply:
@MarianoGnu `is_parent_class` goes recursive on the provided class, starting by itself.
I've tested this code and it works as expected (just like previous version with `ClassDB::is_parent_class(type, "Script")` were hiding `Script` along with `PluginScript`, `GDScript` etc.)

Reply:
and is this ok? won't it hide phython script and nim script?

Reply:
Python script (and nim script, provided it is implemented using pluginscript) won't appear in this list in the first place because it is not a C++ class inheriting from `PluginScript`.

The idea is that a `PluginScript` instance is connected to a `PluginScriptLanguage` which describe the language and it C callback functions to interact with.

That's precisely the reason `PluginScript` is hidden in the first place: if you instantiate it from this dialog box, you cannot configure what language this script is in, so it will crash as soon as you start using it.

Reply:
Well then, i'm probably misunderstanding the use of PluginScript, and it's used to create Languages instead of actual scripts.